.github/workflows/delete-namespace.yml

```diff
@@ -15,22 +15,156 @@ permissions:
 
     
 jobs:
-  delete-namespace:
+  create_keycloak_db_name:
     runs-on: ubuntu-latest
+    outputs:
+      keycloak_db_name: ${{ steps.create_keycloak_db_name.outputs.keycloak_db_name }}
     steps:
-    - name: Check if namespace is allowed, fails if it not starts with ticketnr, e.g. spsh-1234
-      run: | 
-        regex='^([[:alpha:]]+?-[[:digit:]]+)'
-        [[ ${{ inputs.namespace }} =~ $regex ]]
-        echo "${BASH_REMATCH[1]}"
-    - name: Set up kubectl
+      - id: create_keycloak_db_name
+        shell: bash
+        run: |
+          # keycloak db name creation at a single point
+          # Change every "." and "-" into "_", then remove all characters that are not alphanumeric, digit or an underscore and cut it on max lenght for postgresql db name
+          temp=$(echo "spsh_keycloak_${{ inputs.namespace }}" | tr - _ | tr \. _ | sed 's/[^a-zA-Z0-9_]//g' | cut -c1-63)
+          echo "before" ${{ inputs.namespace }} "after" $temp
+          echo "keycloak_db_name=$temp" >> $GITHUB_OUTPUT
+
+  create_spsh_app_db_name:
+    runs-on: ubuntu-latest
+    outputs:
+      spsh_app_db_name: ${{ steps.create_spsh_app_db_name.outputs.spsh_app_db_name }}
+    steps:
+      - id: create_spsh_app_db_name
+        shell: bash
+        run: |
+          # spsh-app db name creation at a single point
+          # Change every "." and "-" into "_", then remove all characters that are not alphanumeric, digit or an underscore and cut it on max lenght for postgresql db name
+          temp=$(echo "spsh_database_${{ inputs.namespace }}" | tr - _ | tr \. _ | sed 's/[^a-zA-Z0-9_]//g' | cut -c1-63)
+          echo "before" ${{ inputs.namespace }} "after" $temp
+          echo "spsh_app_db_name=$temp" >> $GITHUB_OUTPUT^
```

Comment:
Does the ^ at the end of the line do anything?

Reply:
That should not be there, its gone now